orchestrator/src/test/java/com/scottlogic/deg/orchestrator/endtoend/JarExecuteTests.java

```diff
@@ -0,0 +1,28 @@
+package com.scottlogic.deg.orchestrator.endtoend;
+
+import org.junit.jupiter.api.Test;
+import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class JarExecuteTests {
+
+    @Test
+    void GenerateSuccessfullyFromJar() throws Exception {
+            ProcessBuilder pb = new ProcessBuilder("java", "-jar", "build/libs/generator.jar", "generate","-p=src/test/java/com/scottlogic/deg/orchestrator/endtoend/testprofile.profile.json","--max-rows=1");
```

Comment:
Picky formatting comment: please be consistent as to whether or not you put spaces after the commas in the parameter list.  A space is generally preferred, I think